ibm_catalog.json

```diff
@@ -2,7 +2,7 @@
   "products": [
     {
       "name": "deploy-arch-secrets-manager-private-cert-engine",
-      "label": "Cloud automation for Secrets Manager private certificates engine",
+      "label": "Cloud automation for Secrets Manager Private Certificates Engine",
```

Comment:
why are you adding caps to "Private Certificates Engine" ? These should not be capitalised. They are not a service name.